{"name": "The user should be able to check rate revision sucessfully -- @1.1 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=114.0.5735.199)\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x010BA813+48355]\n\t(No symbol) [0x0104C4B1]\n\t(No symbol) [0x00F55358]\n\t(No symbol) [0x00F3D293]\n\t(No symbol) [0x00F9E37B]\n\t(No symbol) [0x00FAC473]\n\t(No symbol) [0x00F9A536]\n\t(No symbol) [0x00F782DC]\n\t(No symbol) [0x00F793DD]\n\tGetHandleVerifier [0x0131AABD+2539405]\n\tGetHandleVerifier [0x0135A78F+2800735]\n\tGetHandleVerifier [0x0135456C+2775612]\n\tGetHandleVerifier [0x011451E0+616112]\n\t(No symbol) [0x01055F8C]\n\t(No symbol) [0x01052328]\n\t(No symbol) [0x0105240B]\n\t(No symbol) [0x01044FF7]\n\tBaseThreadInitThunk [0x76476839+25]\n\tRtlGetFullPathName_UEx [0x7708906F+1215]\n\tRtlGetFullPathName_UEx [0x7708903D+1165]\n\n", "trace": "  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"Features\\environment.py\", line 75, in after_step\n    allure.attach(context.driver.get_screenshot_as_png(), name=\"screenshot\", attachment_type=AttachmentType.PNG)\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 965, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 977, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 444, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 249, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "And click on Rate Revision status", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=114.0.5735.199)\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x010BA813+48355]\n\t(No symbol) [0x0104C4B1]\n\t(No symbol) [0x00F55358]\n\t(No symbol) [0x00F3D293]\n\t(No symbol) [0x00F9E37B]\n\t(No symbol) [0x00FAC473]\n\t(No symbol) [0x00F9A536]\n\t(No symbol) [0x00F782DC]\n\t(No symbol) [0x00F793DD]\n\tGetHandleVerifier [0x0131AABD+2539405]\n\tGetHandleVerifier [0x0135A78F+2800735]\n\tGetHandleVerifier [0x0135456C+2775612]\n\tGetHandleVerifier [0x011451E0+616112]\n\t(No symbol) [0x01055F8C]\n\t(No symbol) [0x01052328]\n\t(No symbol) [0x0105240B]\n\t(No symbol) [0x01044FF7]\n\tBaseThreadInitThunk [0x76476839+25]\n\tRtlGetFullPathName_UEx [0x7708906F+1215]\n\tRtlGetFullPathName_UEx [0x7708903D+1165]\n\n", "trace": "  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"Features\\environment.py\", line 75, in after_step\n    allure.attach(context.driver.get_screenshot_as_png(), name=\"screenshot\", attachment_type=AttachmentType.PNG)\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 965, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 977, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 444, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 249, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1717046724784, "stop": 1717046724788}, {"name": "And change the \"Approved Revision\"", "status": "skipped", "start": 1717046724789, "stop": 1717046724789}, {"name": "And click on save for update", "status": "skipped", "start": 1717046724789, "stop": 1717046724789}, {"name": "And click on Rate Revision status", "status": "skipped", "start": 1717046724789, "stop": 1717046724789}, {"name": "And change the \"Candidate Revision\"", "status": "skipped", "start": 1717046724789, "stop": 1717046724789}, {"name": "And click on save for update", "status": "skipped", "start": 1717046724789, "stop": 1717046724789}, {"name": "And verify the approval history \"Approved Revision\"", "status": "skipped", "start": 1717046724789, "stop": 1717046724789}], "parameters": [{"name": "status", "value": "Approved Revision"}, {"name": "message", "value": "Approved Revision"}, {"name": "status1", "value": "Candidate Revision"}], "start": 1717046723769, "stop": 1717046724789, "uuid": "53106323-cfb1-4ff2-85fd-a6b4b0eb1327", "historyId": "2e1efd0cfe3c66e0ea990f6525c43c33", "fullName": "Verify the TMS functionality: The user should be able to check rate revision sucessfully", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "Rate_Cards_RateRevision"}, {"name": "feature", "value": "Verify the TMS functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}